distributed/client.py

```diff
@@ -677,9 +677,13 @@ def __init__(
             heartbeat_interval = dask.config.get("distributed.client.heartbeat")
         heartbeat_interval = parse_timedelta(heartbeat_interval, default="ms")
 
+        scheduler_info_update = parse_timedelta(
+            dask.config.get("distributed.client.scheduler-info-update")
+        )
```

Comment:
By default `parse_timedelta` will assume an input numeric value is in seconds, so the `* 1000` factor will be off in that case. I think we should specify `default="ms"` here, similar to when we parse `heartbeat_interval`

Reply:
Oh I see. I misunderstood what that default was for.